### ISSUE TITLE
Are you planning to implement CIoU?

### ISSUE THREAD

**Issue by AlexeyAB** [hunglc007/tensorflow-yolov4-tflite#110]
@hunglc007 Hi,
Are you planning to implement CIoU?

Examples: 

* C: https://github.com/AlexeyAB/darknet/blob/a71b9a6e9a009cf94900c53deb344c5204835700/src/box.c#L233-L256

* Matlab: https://github.com/Zzh-tju/DIoU/blob/master/simulation%20experiment/dCIOU.m

* Python: https://github.com/VCasecnikovs/Yet-Another-YOLOv4-Pytorch/blob/2e18612e1852abbf35b4dac55a00f2a3b2d814ed/model.py#L527-L561

* Python: https://github.com/ultralytics/yolov3/blob/eca5b9c1d36e4f73bf2f94e141d864f1c2739e23/utils/utils.py#L262-L282

Desctiption: https://medium.com/@jonathan_hui/yolov4-c9901eaa8e61

![1_egnHdTEZcYJKkcPkDTgaVw](https://user-images.githubusercontent.com/4096485/85800301-626c9400-b749-11ea-93df-6d323f55ce71.jpeg)

**Comment by hunglc007** [comment]
Thank you. Im preparing push a newer version with improvements, which include CIoU loss and yolov4-tiny

**Comment by zcdliuwei** [comment]
> Thank you. Im preparing push a newer version with improvements, which include CIoU loss and yolov4-tiny

the loss is nan, has this problem been solved ?

**Comment by hunglc007** [comment]
> > Thank you. Im preparing push a newer version with improvements, which include CIoU loss and yolov4-tiny
> 
> the loss is nan, has this problem been solved ?

I updated mish activation to fix nan loss issue.

**Comment by zcdliuwei** [comment]
> > > Thank you. Im preparing push a newer version with improvements, which include CIoU loss and yolov4-tiny
> > 
> > 
> > the loss is nan, has this problem been solved ?
> 
> I updated mish activation to fix nan loss issue.

great !!
